lib/ProductOpener/Display.pm

```diff
@@ -274,7 +274,7 @@ $tt = Template->new(
 		EVAL_PERL    => 1,
 		STAT_TTL     => 60,                              # cache templates in memory for 1 min before checking if the source changed
 		COMPILE_EXT  => '.ttc',                          # compile templates to Perl code for much faster reload
-		COMPILE_DIR  => $data_root . '/tmp/templates',
+		COMPILE_DIR  => '/tmp/templates',
```

Comment:
I suggest using a subdir for the server software, like `/tmp/productopener/templates/`.